### PR TITLE
refactor/1780 - implement constants for wake turbulence category & engine type 

### LINKED ISSUE #1780: Constants for wake turbulence category & engine type
Implement constants in `aircraftConstants.js` for wake turbulence category & engine type

Wake turbulence category: resolve todos
https://github.com/openscope/openscope/blob/c391f605530dac4c5e9f8554d4ecf56c5b17f28d/src/assets/scripts/client/aircraft/AircraftTypeDefinitionModel.js#L6-L8
https://github.com/openscope/openscope/blob/c391f605530dac4c5e9f8554d4ecf56c5b17f28d/src/assets/scripts/client/scope/RadarTargetModel.js#L18-L20

Engine type: replace magic literal
https://github.com/openscope/openscope/blob/c391f605530dac4c5e9f8554d4ecf56c5b17f28d/src/assets/scripts/client/aircraft/AircraftModel.js#L797

### PR DESCRIPTION
**Pull request by cake-pie**
Resolves #1780

The purpose of this pull request is to implement constants for wake turbulence category & engine type

### CODE REVIEW

File: src/assets/scripts/client/aircraft/AircraftTypeDefinitionModel.js [hunk 1]
```diff
@@ -196,18 +193,9 @@ export default class AircraftTypeDefinitionModel extends BaseModel {
...
-                break;
-            case HEAVY_LETTER:
-                aircraftIcao = `${HEAVY_LETTER}/${this.icao}/L`;
-
-                break;
-            default:
-
-                break;
+        const wtc = Object.values(WAKE_TURBULENCE_CATEGORY).find((WTC) => WTC.LETTER === this.weightClass) ?? { APPEND: false };
+        if (wtc.APPEND) {
```
Comment (n8rzz) [inline src/assets/scripts/client/aircraft/AircraftTypeDefinitionModel.js:197]: missing empty line above `if`

File: src/assets/scripts/client/scope/RadarTargetModel.js [hunk 2]
```diff
@@ -343,20 +340,11 @@ export default class RadarTargetModel {
...
-            case HEAVY_LETTER:
-                dataBlockRowOne += ` ${HEAVY_LETTER}`;
-
-                break;
-            default:
-
-                break;
+        const wtc = Object.values(WAKE_TURBULENCE_CATEGORY).find((WTC) => WTC.LETTER === this.aircraftModel.model.weightClass) ??
+            { APPEND: false };
+        if (wtc.APPEND) {
```
Comment (n8rzz) [inline src/assets/scripts/client/scope/RadarTargetModel.js:345]: missing empty line above `if`

**Review by n8rzz — APPROVED** [review]

File: src/assets/scripts/client/aircraft/AircraftTypeDefinitionModel.js (hunk 1 above)
Comment (cake-pie) [reply src/assets/scripts/client/aircraft/AircraftTypeDefinitionModel.js:197]: done

File: src/assets/scripts/client/scope/RadarTargetModel.js (hunk 2 above)
Comment (cake-pie) [reply src/assets/scripts/client/scope/RadarTargetModel.js:345]: done

**Pull request merged**